### PR TITLE
feat(dashboard): create admin dashboard

### PR DESCRIPTION
**Pull request by Charpell**
#### What does this PR do?
- Create admin dashboard

#### Description of Task to be completed?
- allow users with admin roles switch to the admin dashboard

#### How should this be manually tested?
- run `npm test` for test and `npm run coverage` to run coverage reports.

#### What are the relevant pivotal tracker stories?
[160051953](https://www.pivotaltracker.com/n/projects/2178749/stories/160051953)

#### Any background context you want to add ?
N/A

#### Screenshots
screen shot 2018-08-29 at 12 46 09 pm
screen shot 2018-08-29 at 12 46 19 pm

### CODE REVIEW

**Comment by Charpell** [comment]
LGTM

**Comment by larrystone** [comment]
Please update the PR with relevant information, e.g description, screenshot, etc

File: client/src/helpers/authorization.js [hunk 1]
```diff
@@ -20,4 +20,12 @@ export const isAuthorized = () => {
...
\ No newline at end of file
+};
+
+
+/**
+ * This function checks if the user 
+ * 
+ * @return {boolean} boolean
+ */
+export const isAdmin = () => (!!decodeToken().roles.Fellow);
```
Comment (larrystone) [inline client/src/helpers/authorization.js]: Does this imply that all Fellows are admins?

Comment (Charpell) [reply client/src/helpers/authorization.js]: No. its for development process

Comment (larrystone) [reply client/src/helpers/authorization.js]: Okay then

**Pull request merged**